Description: replace ava by tape
Author: Yadd <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2022-09-20

--- a/test.js
+++ b/test.js
@@ -1,4 +1,4 @@
-import test from 'ava';
+import test from 'tape';
 import parseMilliseconds from './index.js';
 
 test('parse milliseconds into an object', t => {
@@ -101,6 +101,7 @@
 		microseconds: 0,
 		nanoseconds: 543
 	});
+	t.end();
 });
 
 test('handle negative millisecond values', t => {
@@ -133,5 +134,6 @@
 			t.is(negative[key], -positive[key]);
 		}
 	}
+	t.end();
 });
 
